<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommWindow</class>
 <widget class="QDialog" name="CommWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки связи с ПЛК</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/rs_hmi_logo.png</normaloff>:/rs_hmi_logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QPlainTextEdit" name="msgPlainText">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Droid Sans Mono</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>1.
2.
3.
4.
5.
6.
7.
8.
9.
10.</string>
     </property>
     <property name="maximumBlockCount">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelDevice">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>порт</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBaud">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>частота</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelParity">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>паритет</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBits">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>бит</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelStopBits">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>стоп бит</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDelay">
         <property name="text">
          <string>задержка, мсек</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTimeout">
         <property name="text">
          <string>таймаут, мсек</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBoxPorts"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxBaud"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxParity"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxBits"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxStopBits"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxDelay">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxTimeout">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="commStartButton">
         <property name="text">
          <string>старт обмена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="commStopButton">
         <property name="text">
          <string>стоп обмена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="portOpenButton">
         <property name="text">
          <string>подключить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="portCloseButton">
         <property name="text">
          <string>отключить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendNextButton">
         <property name="text">
          <string>Обновить данные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="rightMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="revertButton">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Исходные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Установить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeWindowButton">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
